src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private int boundNumber = 100;

```

Comment:
It is not a [constant](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)